code/backend/batch/utilities/integrated_vectorization/AzureSearchSkillset.py

```diff
@@ -19,7 +20,7 @@
 
 
 class AzureSearchSkillset:
-    def __init__(self, env_helper: EnvHelper):
+    def __init__(self, env_helper: EnvHelper, integrated_vectorization_config: Config):
```

Comment:
Isn't this a different type now IntegrationVectorizationConfig?

Reply:
Missed checking in the change. Updated now